cmd/instance/instance_update.go

```diff
@@ -60,8 +60,8 @@ If you wish to use a custom format, the available fields are:
 			os.Exit(1)
 		}
 
-		if common.OutputFormat == "human" {
-			fmt.Printf("The instance %s (%s) has been updated\n", utility.Green(instance.Hostname), instance.ID)
+		if common.OutputFormat == common.OutputFormatHuman {
+			fmt.Printf("The instance %s (%s) was updated\n", utility.Green(instance.Hostname), instance.ID)
```

Comment:
why not "has been"?

Reply:
https://github.com/civo/cli/pull/533/commits/1d4b52ae44ad71e53b4f30182f4460eafdb66ebc